public/css/main.css

```diff
@@ -20,14 +19,94 @@ a {
 }
 
 /*** GENERAL ***/
-
 body {
   background-color: #F2F2F2;
 }
 
 /*** HEADER BAR ***/
+.header {
+  background-color: #409A93;
+  font-family: 'Noto Sans', sans-serif;
+  height: 8rem;
+  padding: 2.5rem;
+  z-index: 1;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.naz-events{
+  color: white;
+  font-weight: bold;
+  font-size: 2.3rem;
+}
+
+.calendar-icon {
+  font-size: 2.3rem;
+  color: white;
+  margin-top: .1rem;
+}
+
+.plus-icon{
+  margin-top: .2rem;
+  font-size: 2.3rem;
+  color: white;
+}
+
+.language-nav {
+  background: #F2F2F2;
+  width: 100%;
+  padding: .6rem;
+  box-shadow: .4rem .5rem .5rem #DBDBDB;
+  display: flex;
+}
+
+.lang-btn{
+  width: 48.5%;
+  color: #5C5C5C;
+  padding: .5rem;
+  background: #DBDBDB;
+  border: .1rem solid #DBDBDB;
+  border-radius: .1rem;
+  font: bolder 1.5rem 'Lato', sans-serif;
+  flex-grow: 1;
+}
+
+.en-lang {
+  margin-right: .6rem;
+}
 
 /*** EVENTS LIST ***/
+.container-div {
+  background-color: #EAEAEA;
+  padding: 0.6rem;
+  margin: 1rem;
+  box-shadow: 0 .2rem .2rem #DADADA;
```

Comment:
I think it'd be good to make these a tiny bit bigger - maybe a height of 9rem? They just seem ever so slightly too squashed

Reply:
Then display: flex and align-items: space-between and justify-content: flex-start

Reply:
the flex not doing any thing ? @JWLD